garage.com/src/test/resources/SQL_Basic.txt

```diff
@@ -0,0 +1,57 @@
+Create a new branch called f-sql
+
+Sakila Database
+
+Write a SQL statement for each of the following. 
+
+1. Get 10 cities in descending alphabetical order.
+   SELECT city
+   FROM sakila.city 
+   ORDER BY city DESC
+   LIMIT 10;
+
+2. Get all films with "airplane" in the title.
+  SELECT title
+  FROM `sakila`.`film`
+  WHERE title LIKE('%airplane%');
+
+3. Get the highest payment amount.
+   SELECT max(amount)
+   FROM `sakila`.`payment`;
+
+4. Get the number of records in the customer table for store id #1.
+   SELECT * 
+   FROM sakila.customer 
+   WHERE store_id = '1';
+
+5. Get all payment records for customer with email address "NANCY.THOMAS@sakilacustomer.org"
+  SELECT * 
+  FROM sakila.payment
+  WHERE customer_id = '12';

```

Comment:
We needed to lookup by "NANCY.THOMAS@sakilacustomer.org". 
The quotations hint that we want to lookup by a literal string.